src/components/AvatarWithDisplayName.tsx

```diff
@@ -177,6 +182,7 @@ function AvatarWithDisplayName({
     const ownerPersonalDetails = getPersonalDetailsForAccountIDs(report?.ownerAccountID ? [report.ownerAccountID] : [], personalDetails);
     const displayNamesWithTooltips = getDisplayNamesWithTooltips(Object.values(ownerPersonalDetails), false, localeCompare);
     const avatarBorderColor = avatarBorderColorProp ?? (isAnonymous ? theme.highlightBG : theme.componentBG);
+    const statusText = shouldDisplayStatus && isExpenseReport(report) ? getReportStatusTranslation(report?.stateNum, report?.statusNum) : undefined;
```

Comment:
```suggestion
    const statusText = shouldDisplayStatus && isMoneyRequestReport(report) ? getReportStatusTranslation(report?.stateNum, report?.statusNum) : undefined;
```

We need to show for iou report too bcoz it is shown in all report previews.

Reply:
Yeah, makes sense. They should show up in `outstanding` and `paid` searches after all. 👍